pda.sh

```diff
@@ -0,0 +1,42 @@
+package: PDA
+version: "%(tag_basename)s"
+tag: 11.0.7
+source: https://github.com/PascalBoeschoten/pda
```

Comment:
Correct. 

How should we proceed ? This is a modified fork of https://github.com/cbm-fles/pda which is a fork of https://github.com/kdiff3/pda.